components/Core/VerticalLine/index.tsx

```diff
@@ -0,0 +1,12 @@
+interface VerticalLineProps {
+  height: number,
+  className?: any
+}
+
+export default function VerticalLine({ height, className } : VerticalLineProps) {
+  return (
+    <span
+      className={className}
+      style={{ borderRight: '1px solid black', height: `${height}rem` }} />
```

Comment:
be consistent with styling method

Reply:
The issue is that I need to pass the height value there which to my knowledge cannot be passed into a `.module.css`.